src/bitcoin-cli.cpp

```diff
@@ -738,6 +740,9 @@ static UniValue ConnectAndCallRPC(BaseRequestHandler* rh, const std::string& str
     UniValue response(UniValue::VOBJ);
     // Execute and handle connection failures with -rpcwait.
     const bool fWait = gArgs.GetBoolArg("-rpcwait", false);
+    const int timeout = gArgs.GetArg("-rpcwaittimeout", DEFAULT_WAIT_CLIENT_TIMEOUT);
```

Comment:
nit, is it fine to accept <0? A negative value is same as  `-rpcwait=false`.

Reply:
I think so, the alternative would be to throw an error, but that might be a bit much.